<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>500</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>10000</width>
    <height>500000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>密码操作</string>
  </property>
  <widget class="QWidget" name="widget_add" native="true">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>0</y>
     <width>171</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_add1">
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="text">
       <string>名称</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="line_add1"/>
    </item>
    <item>
     <widget class="QLabel" name="label_add2">
      <property name="text">
       <string>用户名</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="line_add2"/>
    </item>
    <item>
     <widget class="QLabel" name="label_add3">
      <property name="text">
       <string>密码</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="line_add3"/>
    </item>
    <item>
     <widget class="QLabel" name="label_add4">
      <property name="text">
       <string>备注 </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="line_add4"/>
    </item>
    <item>
     <widget class="QPushButton" name="button_add2">
      <property name="text">
       <string>添加</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>77</width>
     <height>501</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="button_add">
      <property name="text">
       <string>添加密码</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_del">
      <property name="text">
       <string>删除密码</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_change">
      <property name="text">
       <string>查询密码</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_search">
      <property name="text">
       <string>修改密码</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
